Resources/views/Import/select.html.twig

```diff
@@ -3,10 +3,22 @@
 {% block content %}
     <div class="list-group">
         {% for importer in importers %}
-            <a href="{{ path('cmfcmfmediamodule_import_execute', {importer: importer.id}) }}" class="list-group-item">
-                <h4 class="list-group-item-heading">{{ importer.title }}</h4>
-                <p class="list-group-item-text">{{ importer.description }}</p>
-            </a>
+            {% set requirements = importer.checkRequirements %}
+            {% if requirements is same as(true) %}
```

Comment:
No, because [`true == "foo"`](https://3v4l.org/8mb6T) in which case there is an unfulfilled requirement.